Javadoc warnings with Java 17

```
[INFO] --- maven-javadoc-plugin:3.4.1:jar (default) @ gitflow-incremental-builder ---
[INFO] No previous run data found, generating javadoc.
[WARNING] Javadoc Warnings
[WARNING] Loading source files for package io.github.gitflowincrementalbuilder.jgit...
[WARNING] Loading source files for package io.github.gitflowincrementalbuilder...
[WARNING] Loading source files for package io.github.gitflowincrementalbuilder.config...
[WARNING] Loading source files for package io.github.gitflowincrementalbuilder.mojo...
[WARNING] Constructing Javadoc information...
[WARNING] Building index for all the packages and classes...
[WARNING] Standard Doclet version 17.0.5+8-LTS
[WARNING] Building tree for all the packages and classes...
[WARNING] Generating /home/moldowan/proj/gitflow-incremental-builder/target/apidocs/io/github/gitflowincrementalbuilder/config/Configuration.html...
[WARNING] /home/moldowan/proj/gitflow-incremental-builder/src/main/java/io/github/gitflowincrementalbuilder/config/Configuration.java:34: warning: no comment
[WARNING] public class Configuration {
[WARNING] ^
...
```

--- a/pom.xml
+++ b/pom.xml
@@ -343,6 +343,9 @@
                     <artifactId>maven-javadoc-plugin</artifactId>
                     <version>${version.maven-javadoc-plugin}</version>
                     <configuration>
+                        <!-- exclude 'missing' from doclint for now (avoids 'warning: no comment' spam), see also:
+                             https://docs.oracle.com/en/java/javase/17/docs/specs/man/javadoc.html#additional-options-provided-by-the-standard-doclet -->
+                        <doclint>accessibility,html,reference,syntax</doclint>
                         <source>${maven.compiler.source}</source>
                     </configuration>
                 </plugin>

